README.md

```diff
@@ -1,112 +1,20 @@
-# Akka-gRPC Lagom Quickstart (scala)
+# Lagom Akka gRPC Quickstart with Scala
```

Comment:
I moved a lot of text from `README` into `docs/src/main/paradox/index.md` below, this is why this edit is so huge. The important thing is the final look, rather than the diff.